lib/cache/cache_test.go

```diff
@@ -602,7 +602,7 @@ func TestCompletenessInit(t *testing.T) {
 
 	// put lots of CAs in the backend
 	for i := 0; i < caCount; i++ {
-		ca := suite.NewTestCA(types.UserCA, fmt.Sprintf("%d.example.com", i))
+		ca := suite.NewTestCA(types.UserCA, fmt.Sprintf("%d.example.org", i))
```

Comment:
Is this change needed?

Reply:
Not really
It was only meant for the screenshot, but should do no harm
Reverting anyway :+1: